libs/core/src/main/java/org/opensearch/core/common/util/CollectionUtils.java

```diff
@@ -140,7 +140,10 @@ private static Iterable<?> convert(Object value) {
         }
         if (value instanceof Map) {
             Map<?, ?> map = (Map<?, ?>) value;
-            return () -> Iterators.concat(map.keySet().iterator(), map.values().iterator());
+            Iterator<?>[] iterators = new Iterator<?>[2];
```

Comment:
Let's just add `@SuppressWarnings("unchecked")` to be consistent w/ [the 2.x branch](https://github.com/opensearch-project/OpenSearch/blob/d090a7f1e8be0df161f3d48f264bd0943766417e/libs/core/src/main/java/org/opensearch/core/common/util/CollectionUtils.java#L137).

Reply:
Changed to use `@SuppressWarnings("unchecked")`